packages/client/hmi-client/src/components/widgets/tera-signal-bars.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="flex align-items-center gap-2">
```

Comment:
You can move the following into a css selector `div` like you did for the rest of the component.